nnpdf_data/nnpdf_data/commondata/ATLAS_Z0_8TEV_LOWMASS/filter.py

```diff
@@ -0,0 +1,93 @@
+"""
+filter.py module for ATLAS_Z0_8TEV_LOWMASS dataset
+When running `python filter.py` the relevant uncertainties , data and kinematics yaml
+file will be created in the `nnpdf_data/commondata/ATLAS_Z0_8TEV_LOWMASS` directory.
+"""
+
+import yaml
+from filter_utils import get_kinematics, get_data_values, get_systematics
+from nnpdf_data.filter_utils.utils import prettify_float
+
+yaml.add_representer(float, prettify_float)
+
+
+def filter_ATLAS_Z0_8TEV_LOWMASS_data_kinetic():
+    """
+    This function writes the central values and kinematics to yaml files.
+    """
+
+    kin = get_kinematics()
+    central_values = list(get_data_values())
+
+    data_central_yaml = {"data_central": central_values}
+
+    kinematics_yaml = {"bins": kin}
+
+    # write central values and kinematics to yaml file
+    with open("data.yaml", "w") as file:
+        yaml.dump(data_central_yaml, file, sort_keys=False)
+
+    with open("kinematics.yaml", "w") as file:
+        yaml.dump(kinematics_yaml, file, sort_keys=False)
+
+
+def filter_ATLAS_Z0_8TEV_LOWMASS_systematics(version=3):
+    """
+    This function writes the systematics to a yaml file.
+    """
+
+    with open("metadata.yaml", "r") as file:
+        metadata = yaml.safe_load(file)
+
+    systematics = get_systematics(version=version)
+
+    # error definition
+    error_definitions = {}
+    errors = []
+
+    for sys in systematics:
+        if (sys[0]['name'] == 'stat') or (sys[0]['name'] == 'sys,uncor'):
+            error_definitions[sys[0]['name']] = {
+                "description": f"{sys[0]['name']}",
+                "treatment": "ADD",
+                "type": "UNCORR",
+            }
+
+        elif (sys[0]['name'] == 'ATLAS_LUMI') or (sys[0]['name'] == 'Lumi:M'):
+            error_definitions[sys[0]['name']] = {
+                "description": f"{sys[0]['name']}",
+                "treatment": "MULT",
+                "type": "UNCORR",
+            }
+
```

Comment:
A possible bug here is that the LUMI in the legacy implementation was treated as UNCORR.
It should however be CORR (ATLASLUMI12). If I set it to CORR, however, the chi2 changes by an order of magnitude (see report)
Am I missing something here or is it genuine?


https://vp.nnpdf.science/X59c09ozTEmm9vwP3vYTRg==